app/models/concerns/foreman_bootdisk/orchestration/compute.rb

```diff
@@ -85,6 +96,14 @@ def rebuild_with_bootdisk
           return false
         end
       end
+
+      private
+      def is_rebuild?
```

Comment:
Nitpick: This is concern is included into Host::Managed, e.g. this will provide `host.is_rebuild?`.  Maybe something like `host.left_build_mode?` or similar?

Reply:
I had the same idea. I added this to foreman_bootdisk to finish this task sooner. :-)

Reply:
it can stay in the plugin, I just think the naming is misleading.